controller/UserController.js

```diff
@@ -0,0 +1,103 @@
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const jwt = require("jsonwebtoken");
+const crypto = require("crypto");
+
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+const join = (req, res) => {
+  const { email, password } = req.body;
+  const salt = crypto.randomBytes(10).toString("base64");
+  const hashPassword = crypto
+    .pbkdf2Sync(password, salt, 10000, 10, "sha512")
+    .toString("base64");
```

Comment:
```suggestion
  const salt = crypto.randomBytes(10).toString("base64");
  const hashPassword = crypto
    .pbkdf2Sync(password, salt, 10000, 10, "sha512")
    .toString("base64");
```
암호화 하는 부분은 함수로 따로 분리하는 것이 좋을 것 같습니다.